test/generators/java/presets/CommonPreset.spec.ts

```diff
@@ -29,6 +29,27 @@ describe('JAVA_COMMON_PRESET', () => {
     const classModel = await generator.renderClass(model, inputModel);
     expect(classModel.result).toMatchSnapshot();
   });
+  test('should not render any functions when all 3 options are disabled', async () => {
+    const options: JavaCommonPresetOptions = {
+      equal: false,
+      hashCode: false,
+      classToString: false,
+    };
+
+    const generator = new JavaGenerator(
+      {
+        presets: [{
+          preset: JAVA_COMMON_PRESET,
+          options
+        }]
+      }
+    );
+    const inputModel = await generator.process(doc);
+    const model = inputModel.models['Clazz'];
+
+    const classModel = await generator.renderClass(model, inputModel);
+    expect(classModel.result).toMatchSnapshot();
+  });
```

Comment:
@kamko do you mind moving this test into the [related group of tests for options](https://github.com/asyncapi/modelina/blob/cf3658dd55901042f12fbc3ddc91e2d483b93a9d/test/generators/java/presets/CommonPreset.spec.ts#L53)? 🙂

Reply:
I had it there first but then I moved it to the top level as I'm only testing that it will compile and not the options themselves.

Although a second look tells me that there wasn't the test for the all-false case so it fits there 😅